packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -86,8 +92,80 @@ class CupertinoSlider extends StatefulWidget {
   ///   },
   /// )
   /// ```
+  /// 
+  /// See also:
+  ///
+  ///  * [onChangeStart] for a callback that is called when the user starts
+  ///    changing the value.
+  ///  * [onChangeEnd] for a callback that is called when the user stops
+  ///    changing the value.
   final ValueChanged<double> onChanged;
 
+  /// Called when the user starts selecting a new value for the slider.
+  ///
+  /// This callback shouldn't be used to update the slider [value] (use
+  /// [onChanged] for that), but rather to be notified when the user has started
+  /// selecting a new value by starting a drag.
+  ///
+  /// The value passed will be the last [value] that the slider had before the
+  /// change began.
+  ///
+  /// ## Sample code
+  ///
+  /// ```dart
+  /// new CupertinoSlider(
+  ///   value: _duelCommandment.toDouble(),
```

Comment:
You'll need to add the magic comment incantation:

```dart
// Examples can assume:
// int _duelCommandment = 1;
```
at the top of the file, just below the imports (see the top of material/slider.dart for an example) to prevent the dartdoc syntax checker from failing.

Reply:
Done :)